platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/protocol/SyncProtocol.java

```diff
@@ -173,19 +156,14 @@ public boolean shouldInitiate() {
             return false;
         }
 
-        // is there a reason to initiate?
-        if (!useCriticalQuorum || peerNeededForFallenBehind() || criticalQuorum.isInCriticalQuorum(peerId)) {
-            final boolean isLockAcquired = permitProvider.tryAcquire(peerId);
-
-            if (isLockAcquired) {
-                syncMetrics.updateSyncPermitsAvailable(permitProvider.getNumAvailable());
-                syncMetrics.outgoingSyncRequestSent();
-            }
```

Comment:
This could potentially negatively affect performance if we shun slow nodes. @alittley wdyt?

Reply:
@lpetrovic05 By shunning slow nodes, are you referring to sync++?

Reply:
There was logic here to pick who to sync with based on CQ that would pick nodes whose events we don't have. But I see now that this has been disabled, so no negative consequences I guess.